### PR TITLE
Add CRUD routes

### PR DESCRIPTION
**Pull request by wzhao830**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -31,6 +30,24 @@ let returnData = (id) => {
...
+  return Information.deleteOne({
+    productId: id
+  }).exec();
+};
 
 module.exports.returnData = returnData;
 module.exports.Information = Information;
+module.exports.createInformation = createInformation;
+module.exports.updateInformation = updateInformation;
+module.exports.deleteInformation = deleteInformation;
```
Comment (hannahmanfredi) [inline database/index.js:53]: Great job adding CRUD routes! I'm not too familiar with your service but wondering if in your createInfo. method you meant to use .insertMany vs. .insertOne? It depends on your use-case but just curious why you made that decision.

File: server/crudtest.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+const axios = require('axios');
```
Comment (hannahmanfredi) [inline server/crudtest.js:1]: delete commented out code for readability?

File: src/app.jsx [hunk 3]
```diff
@@ -15,8 +15,8 @@ class App extends React.Component {
       avgReviews: 0,
       totalReviews: 0
     };
-    this.informationip = env.INFORMATION_IP || localhost;
-    this.reviewsip = env.REVIEW_IP || localhost;
+    this.informationip = /* env.INFORMATION_IP || */ 'localhost';
```
Comment (hannahmanfredi) [inline src/app.jsx:18]: what does this line do?

File: src/app.jsx [hunk 4]
```diff
@@ -33,16 +33,16 @@ class App extends React.Component {
...
-      .then((response) =>
-        that.setState({
-          avgReviews: response.data.averageReviews,
-          totalReviews: response.data.totalReviews
-        })
-      )
-      .catch((error) =>
-        console.log('ERROR GETTING REVIEW', error)
-      );
+    // axios.get(`http://${this.reviewsip}:3004/averagereview/` + this.state.id)
```
Comment (hannahmanfredi) [inline src/app.jsx:36]: looks good but why is it commented out?

**Review by hannahmanfredi — APPROVED** [review]
Great job adding your CRUD routes! I'd love to see some tests with this PR.

File: database/index.js (hunk 1 above)
Comment (wzhao830) [reply database/index.js:53]: I ran into an issue using .insertOne so I opted to use .insertMany -- I will refactor this once I've resolved the issue.

File: server/crudtest.js (hunk 2 above)
Comment (wzhao830) [reply server/crudtest.js:1]: This is a small file that I created to test the endpoints -- I will add tests soon.

File: src/app.jsx (hunk 3 above)
Comment (wzhao830) [reply src/app.jsx:18]: This service pulls from other services. I decided to comment out the external service request temporarily. I found that the env object wasn't working as intended, so I just set it to localhost for testing purposes.

File: src/app.jsx (hunk 4 above)
Comment (wzhao830) [reply src/app.jsx:36]: This is calling an external service which is no longer running.

**Pull request merged**